internal/walk/walklinks.go

```diff
@@ -24,7 +24,11 @@ func pathRewriter(src string, dest string, fn fs.WalkDirFunc) fs.WalkDirFunc {
 // Determines if path matches pattern, striving for parity with rsync,
 // ("Include/Exclude Pattern Rules", https://linux.die.net/man/1/rsync).
 func matchPattern(path string, pattern string) (bool, error) {
-	if strings.ContainsAny(pattern, "*?") {
+	// This pattern is special; the desired outcome is that
```

Comment:
Why do you say it's special?  It doesn't seem like any special algorithm should be needed to handle it.  It should be possible for the rest of the code to handle this as well as similar patterns without having to add a dedicated path for this pattern.  e.g. "a.*" should be able to match a value of "a.out".

Reply:
The problem is this; `rsync --exlcude='.*'` will exclude "dir/.conf" but not "dir/some.conf". This sure seems like unique behavior to me, and we need to make exodus-rsync emulate it. The obvious solution (and only one I've been able to come up with) is to anchor that `.` but we can't just prefix every pattern with a carat.
This solution still permits matching files like "a.out" with patterns like "a.*". Only if the pattern is exactly `.*` do we know the intent is to match files beginning with `.`.